src/Integration.UnitTests/Connection/ConnectionWorkflowTests.cs

```diff
@@ -227,77 +204,6 @@ public async Task ConnectionWorkflow_ConnectionStep_Credentials_Invalid()
             AssertCredentialsNotStored(); // Connection was rejected by SonarQube
         }
 
-        [TestMethod]
-        public async Task ConnectionWorkflow_ConnectionStep_WhenMissingCSharpPluginAndVBNetPlugin_AbortsWorkflowAndDisconnects()
```

Comment:
I deleted all of the tests that depended on the plugins available on the server.